src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar-start-day--mon calendar-month-length--31">
```

Comment:
we need to fix modifiers naming according to the [convention](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem)